test/fixtures/es-module-loaders/register-loader-app.mjs

```diff
@@ -0,0 +1 @@
+console.log('running application')
```

Comment:
Why not reuse existing fixtures? You could reimplement some of the existing loaders tests (and fixtures) to happen via `registerLoader()` instead of the `--loader` flag. Your test could confirm the same expected results as the original tests.

Reply:
I will try that. But at this point is not possible to use the `registerLoader` function without the `--experimental-loader`/`--loader` flag. If the flag does not exist, the `DefaultModuleLoader` will be used instead of the `CustomModuleLoader`. I'm not focusing on fixing it right now because this feature will still need some `--experimental...` flag to be used. It would be very good if I could use the same `--experimental-loader` flag to do that, but I don't know how to remove the "required argument" option from it.